pkgs/development/python-modules/pyscard/default.nix

```diff
@@ -37,36 +37,31 @@ buildPythonPackage rec {
   nativeCheckInputs = [ pytestCheckHook ];
 
   postPatch =
-    if withApplePCSC then
-      ''
-        substituteInPlace smartcard/scard/winscarddll.c \
-          --replace-fail "/System/Library/Frameworks/PCSC.framework/PCSC" \
-                    "${PCSC}/Library/Frameworks/PCSC.framework/PCSC"
-      ''
-    else
-      ''
-        substituteInPlace setup.py --replace "pkg-config" "$PKG_CONFIG"
-        substituteInPlace smartcard/scard/winscarddll.c \
-          --replace-fail "libpcsclite.so.1" \
-                    "${lib.getLib pcsclite}/lib/libpcsclite${stdenv.hostPlatform.extensions.sharedLibrary}"
-      '';
+    ''
+      substituteInPlace pyproject.toml \
+        --replace-fail 'requires = ["setuptools","swig"]' 'requires = ["setuptools"]'
+    ''
```

Comment:
I did make a python swig package, we could use that instead of this replace, although both will work

Reply:
That package is mostly unnecessary, since pyscard only runs the swig binary to generate the bindings and does not actually depend on the python package.

Reply:
true, that's why I made it using the replace originally